react/src/js/components/Consonant/Container/Container.jsx

```diff
@@ -443,7 +443,7 @@ const Container = (props) => {
             const chFilter = key.toLowerCase().replace('ch_', '').replace(' ', '-');
             if (key.indexOf(filterGroupPrefix) !== 0
                 && !id.toLowerCase().includes(chFilter)
-                || !group.toLowerCase().includes(chFilter)) {
+                || !group.toLowerCase().replace(' ', '-').includes(chFilter)) {
```

Comment:
This will replace only the first space, you should use replaceAll(),
Try it in the console
`'a b c d'.replace(' ', '-')` vs. `'a b c d'.replaceAll(' ', '-')`

Reply:
good catch